src/vl-map-search.js

```diff
@@ -123,50 +129,112 @@ export class VlMapSearch extends vlElement(HTMLElement) {
       this.__searchEventListenerRegistered = true;
       this._selectElement.addEventListener('search', (event) => {
         if (event && event.detail && event.detail.value) {
-          fetch(this.searchUrl + event.detail.value).then((response) => {
-            return response.json();
-          }).then((data) => {
-            if (data && data.SuggestionResult) {
-              const resultaten = data.SuggestionResult.map((resultaat) => {
-                return {
-                  value: resultaat,
-                  label: resultaat,
-                };
-              });
-              this._selectElement.choices = resultaten;
-            }
-          });
+          const lambertCoordinaat = LambertCoordinaat.of(event.detail.value);
+
+          if (LambertCoordinaat.isValid(lambertCoordinaat)) {
+            this._searchChoicesByLambertCoordinaat(lambertCoordinaat);
+          } else {
+            this._searchChoicesByValue(event.detail.value);
+          }
         }
       });
     }
   }
 
+  _searchChoicesByValue(searchValue) {
+    fetch(new URL(this.searchUrl + searchValue).toString()).then((response) => {
+      return response.json();
+    }).then((data) => {
+      if (data && data.SuggestionResult) {
+        const resultaten = data.SuggestionResult.map((resultaat) => {
+          return {
+            value: resultaat,
+            label: resultaat,
+          };
+        });
+        this._selectElement.choices = resultaten;
+      }
+    });
+  }
+
+  _searchChoicesByLambertCoordinaat(lambertCoordinaat) {
+    const encodedUrl = encodeURI(new URL(this.locationXyUrl + lambertCoordinaat.x + ',' + lambertCoordinaat.y));
```

Comment:
Denk dat encodeURI hier niet nodig is. Als ge al iets encode zou ik enkel lambertCoordinaat.x en lamberCoordinaat.y encoden met encodeURIComponent. 
Maar aangezien lambertCoordinaat numbers bevat, denk ik niet dat dat nodig is.

Reply:
Decimale nummers wel ...

Reply:
```
> encodeURIComponent(123.1)
"123.1"
```

Een decimaal getal hoeft dus niet encoded te worden.